examples/pxScene2d/src/pxShaderResource.cpp

```diff
@@ -390,18 +399,27 @@ void pxShaderResource::loadResourceFromFile()
       break;
     }
 
-    loadVtxShader = rtLoadFile(mVertexUrl, mVertexSrc);
+    bool isVtxFILE =  (  mVertexUrl.beginsWith("file://") ||   mVertexUrl.beginsWith("file://") );
+    rtString pathVtx = mFragmentUrl;
```

Comment:
This should be `rtString pathVtx = mVertexUrl;`

Reply:
oops

Reply:
Fixed